packages/pixeloven/cli-addon-storybook/src/extensions/storybook.ts

```diff
@@ -1,27 +1,37 @@
 import storybook from "@storybook/react/standalone";
 import {
-    AddonStorybookRunContext,
+    AddonStorybookToolbox,
+    StorybookExecutionOptions,
     StorybookExecutionType,
     StorybookExtension,
 } from "../types";
 
-export default (context: AddonStorybookRunContext) => {
-    const extension: StorybookExtension = async (
+export default (toolbox: AddonStorybookToolbox) => {
+    const story: StorybookExtension = async (
         type: StorybookExecutionType,
-        args: string[] = [],
+        options: StorybookExecutionOptions,
     ) => {
-        const { filesystem, print } = context;
+        const { filesystem, print } = toolbox;
         const configEntryPoint = require.resolve("@pixeloven-storybook/config");
         const configDir = filesystem.path(configEntryPoint, "..");
+        const {outputDir, port, noQuiet} = options;
+
         try {
             switch (type) {
                 case StorybookExecutionType.build: {
-                    const outputDir = "./dist/public/docs";
+                    /**
+                     * [x] configure output, port, quiet
+                     * [x] include feedback validation on bad values if possible
+                     * [x] - look into adding exit code to invalidargument in toolbox
+                     * [x] - look at webpack cli for examples of erroring
+                     * [ ] - put logic in extensions vs commands being dumb
+                     */
+
                     await storybook({
                         configDir,
                         mode: "static",
-                        outputDir,
-                        quiet: true,
+                        outputDir: outputDir || "./dist/public/docs",
```

Comment:
I think it might be better to not put this in the public directory anymore.For the other CLI output paths we do 
```
coverage/
dist/
docs/
stats/
...
```
should we do a stories/ as the default?

Reply:
`stories/` makes sense to me